src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -6742,6 +6742,40 @@
                       }
                     }
                   }
+                },
+                "securityResponseHeaders": {
+                  "properties": {
+                    "strictTransportSecurity": {
+                      "type": "keyword",
+                      "_meta": {
+                        "description": "The strictTransportSecurity response header, if it is configured."
```

Comment:
> if it is configured

I don't think this is true. We are always reporting it. Either the configured value or `"NULL"`. Maybe I misread the code though. Can you confirm @jportner?

Reply:
Oh, yes sorry about that, I forgot to update the descriptions when I added the `null` coalescing. I'll update those now.